Frameworks/CoreText/CTLine.mm

```diff
@@ -48,16 +60,7 @@ - (instancetype)copyWithZone:(NSZone*)zone {
  @Status Stub
 */
 CTLineRef CTLineCreateWithAttributedString(CFAttributedStringRef string) {
-    UNIMPLEMENTED();
-    NSString* str = [(NSAttributedString*)string string];
-    CFRange lineRange;
-    lineRange.location = 0;
-    lineRange.length = [str length];
-
-    _CTLine* line = [_CTLine new];
-    line->_strRange = lineRange;
-
-    return (CTLineRef)line;
+    return static_cast<CTLineRef>(_DWriteGetLine(string));
```

Comment:
discussed offline, reference platform returns an empty object when given an empty string

---
In reply to: [79685344](https://github.com/Microsoft/WinObjC/pull/989#discussion_r79685344) [](ancestors = 79685344)